blueocean-dashboard/src/main/js/Dashboard.jsx

```diff
@@ -12,7 +12,7 @@ appConfig.loadConfig();
 sse.connect({
     clientId: 'jenkins_blueocean',
     onConnect: undefined,
-    jenkinsUrl: `${appConfig.getJenkinsRootURL()}/`, // FIXME sse should not require this to end with a /
```

Comment:
@tfennelly the purpose of this PR is to fix this hack, but needs the corresponding change in SSE, which hasn't been merged AFAIK